columnflow/tasks/cutflow.py

```diff
@@ -190,7 +200,7 @@ def get_plot_parameters(self):
         params = super().get_plot_parameters()
 
         # no ratio plot implemented: disable ratio plot for `plot_cutflow`
-        if self.plot_function_name == "columnflow.plotting.example.plot_cutflow":
+        if self.plot_function_name_1d == "columnflow.plotting.example.plot_cutflow":
```

Comment:
Maybe `skip_ratio` should rather become a parameter? Having plot function dependent settings somehow works around the function plug-in mechanism and might not scale too well.